trading/models.py

```diff
@@ -92,14 +94,27 @@ def trade_stock(self, stock, quantity):
         """
         Trades a stock for the account
         """
-        if self.has_enough_cash(stock.latest_quote() * quantity) and (
-                self.has_enough_stock(stock, quantity)):
+        if self.has_enough_cash(stock.latest_quote().value * quantity) and (
+                self.has_enough_stock(stock, -1 * quantity)):
             return self.trades.create(
                 quantity=quantity,
                 stock=stock,
             )
         raise Exception("You don't have the necessary resources!")
 
+    def available_cash(self):
+        """
+        Returns the available cash for the trading account
+        """
+        return (
+            self.trading_balance() +
```

Comment:
shouldn't be + trading_balance, should be current_balance() - trading_balance at tradetime.  I think more like: https://github.com/Neitsch/ASE4156/pull/97/files#diff-17490ab817133d204b65862e5d47c947R133

Reply:
Discussed offline - it is correct